CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # v6
 - The `@agent` macro has been rewritten to support fields with default and const values. The old version still works but it's deprecated. Since now the macro supports these features, using `@agent` is the only supported way to create agent types for Agents.jl.
-- The `add_agent!` function supports adding an agent propagating keywords arguments. `add_agent!(agent::AbstractAgent, pos::ValidPos, model::ABM)` and `add_agent!(agent::AbstractAgent, model::ABM)` have been deprecated to allow controlling the ids given to the agents, see issue #861.
+- Manually setting or altering the ids of agents is no longer allowed. The agent id is now considered a read-only field, and is set internally by Agents.jl to enable hidden optimizations in the future. As a consequence, 
+`add_agent!(agent::AbstractAgent, pos::ValidPos, model::ABM)`  and `add_agent!(agent::AbstractAgent, model::ABM)`  have been deprecated. See issue #861 for more.
```

Comment:
The formatting is off here. You are startng a new line without identation, i am not sure this will register as part of the bullet point. better to not start a new line alltogether.